terraform/modules/env-dashboards/upload-dashboard.tf

```diff
@@ -20,6 +20,502 @@ locals {
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [
+        {
+          "alias": "5XX Errors",
+          "yaxis": 2
+        },
+        {
+          "alias": "4XX Errors",
+          "yaxis": 2
+        }
+      ],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "alias": "Count",
+          "dimensions": {
+            "ApiName": "upload.lambdas.api_server",
+            "Stage": "${var.env}"
+          },
+          "hide": false,
+          "highResolution": false,
+          "metricName": "Count",
+          "namespace": "AWS/ApiGateway",
+          "period": "",
+          "refId": "A",
+          "region": "default",
+          "statistics": [
+            "Sum"
+          ]
+        },
+        {
+          "alias": "4XX Errors",
+          "dimensions": {
+            "ApiName": "upload.lambdas.api_server",
+            "Stage": "${var.env}"
+          },
+          "hide": false,
+          "highResolution": false,
+          "metricName": "4XXError",
+          "namespace": "AWS/ApiGateway",
+          "period": "",
+          "refId": "B",
+          "region": "default",
+          "statistics": [
+            "Sum"
+          ]
+        },
+        {
+          "alias": "5XX Errors",
+          "dimensions": {
+            "ApiName": "upload.lambdas.api_server",
+            "Stage": "${var.env}"
+          },
+          "hide": false,
+          "highResolution": false,
+          "metricName": "5XXError",
+          "namespace": "AWS/ApiGateway",
+          "period": "",
+          "refId": "C",
+          "region": "default",
+          "statistics": [
+            "Sum"
+          ]
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Api Gateway Stats [Rate]",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "account-cloudwatch",
```

Comment:
I know that this is unlikely to change, but I'd parameterize all occurences of `account-cloudwatch` to `${var.aws_cloudwatch_data_source_name}` anyway so that your dashboard doesn't break if the datasource name changes.